src/components/shared/VirtualList.js

```diff
@@ -129,7 +129,7 @@ class VirtualListInner extends React.PureComponent<VirtualListInnerProps> {
         className={className}
         ref={this._takeContainerRef}
         style={{
-          height: `${items.length * itemHeight}px`,
+          height: `${items.length * itemHeight + 20}px`,
```

Comment:
Please add a comment here explaining that padding is being added to the list height to take into account the overlay scrollbars.